src/WorkBC.Web/Services/RecommendedJobsService.cs

```diff
@@ -166,6 +167,13 @@ public async Task<AccountCriteria> GetUserAccountCriteria(JobSeeker jobSeeker)
                 .OrderByDescending(a => a.Count)
                 .ToDictionary(k => k.Term, v => v.Count);
 
+            // group the saved jobs by noc code 2021
```

Comment:
Looks good, but do we need to keep the processing for `NocCodes` as well? right above this block.

Reply:
No, we don't need to, but we have taken the approach everywhere else to add Noc 2021 code logic and leave Noc 2016 as is. I can remove and test it if you approve.

Reply:
Yes, but in this case we're doing an additional query for the NOC 2016 which we're no longer using... maybe best to comment out the query and ensure the code doesn't break.

Reply:
The UI display breaks when I comment out the Noc Code query part. I am investigating the part where the UI might be dependent on the Noc code data.

Reply:
Don't spend too much time on this - it's more important to solve actual bugs :-D